plugins/modules/aix_devices.py

```diff
@@ -108,7 +108,7 @@
     device: en1
     attributes:
       mtu: 900
-      arp: off
+      arp: false
```

Comment:
I think this should be string, just like in the [integration test](https://github.com/ansible-collections/community.general/pull/6152/files#diff-13e125ace2da722b0ad5442bbcfbab8c3079eb02c553dd0c65316775de78c8e6R64) (or if i'm wrong, the integration test needs a change)
```suggestion
      arp: 'off'
```

Reply:
You are right, this should (very likely) be a string. Thanks!